README.md

```diff
@@ -1026,6 +1072,275 @@ export default class Page extends React.PureComponent {
 
 This `<Navbar>` will show the user's blockie and mana balance because it is connected to the store.
 
+## Navbar
+
+The `<Navbar>` container can be used in the same way as the `<Navbar>` component from `decentaland-ui` but it's already connected to the redux store. You can override any `NavbarProp` if you want to connect differently, and you can pass all the regular `NavbarProps` to it.
```

Comment:
Maybe we should add a link to `[decentraland-ui](https://github.com/decentraland/ui)` here and in the other instances?

Reply:
makes sense, I'll add those 👍